components/arm/fake/fake.go

```diff
@@ -32,17 +42,41 @@ var fakeModelJSON []byte
 
 // AttrConfig is used for converting config attributes.
 type AttrConfig struct {
-	FailNew      bool   `json:"fail_new"`
-	FailValidate bool   `json:"fail_validate"`
-	ArmModel     string `json:"arm-model"`
+	ArmModel  string `json:"arm-model,omitempty"`
+	ModelPath string `json:"model-path,omitempty"`
+}
+
+func modelFromName(model, name string) (referenceframe.Model, error) {
+	switch resource.ModelName(model) {
+	case xarm.ModelName6DOF.Name:
+		return xarm.Model(name, 6)
+	case xarm.ModelName7DOF.Name:
+		return xarm.Model(name, 7)
+	case ur.ModelName.Name:
+		return ur.Model(name)
+	case yahboom.ModelName.Name:
+		return yahboom.Model(name)
+	case eva.ModelName.Name:
+		return eva.Model(name)
+	case ModelName.Name:
```

Comment:
@biotinker this is checking for the modelName, so a user could access the default json if they pass "fake"

Reply:
I think theres no situation where we want to allow the user access to that default kinematics file and that it should be purely possible from the backend

Reply:
I can get behind that.

If we do this a bunch of tests will have to be somewhat redesigned